solidity/depool/README.md

```diff
@@ -804,6 +831,55 @@ Where
 
 `<msig_address>` - address of the wallet that made the stake.
 
+Output example:
+
+```bash
+Result: {
+  "locks": {
+    "0x54": {
+      "amount": "0x5af32e47a500",
+      "lastWithdrawalTime": "0x5fa517f7",
+      "owner": "0:de7dc920ff73ae077035f26715986aa8039eadbec25f525a0b8fb15dfc74ad0b",
+      "withdrawalPeriod": "0x1a5e00",
+      "withdrawalValue": "0x5af32e47a500"
+    },
+    "0x55": {
+      "amount": "0x5af32e47a500",
+      "lastWithdrawalTime": "0x5fa517f7",
+      "owner": "0:de7dc920ff73ae077035f26715986aa8039eadbec25f525a0b8fb15dfc74ad0b",
+      "withdrawalPeriod": "0x1a5e00",
+      "withdrawalValue": "0x5af32e47a500"
+    }
+  },
+  "reinvest": true,
+  "reward": "0x106dbf302591",
+  "stakes": {
+    "0x54": "0x51779432da5",
+    "0x55": "0x72653cb14c1ec"
+  },
+  "total": "0x7e151a0e73991",
+  "vestings": {},
+  "withdrawValue": "0x0"
+}
+```
+
+The participant parameters displayed by the get-method are the following:
+
+`locks` and `vestings`: participant's lock and vesting stake, each split into two neighboring rounds. There can be only one of each, split equally into two entries. The parameters of the lock and vesting stake are:
```

Comment:
typo `stakes`

Reply:
fixed

Reply:
I meant `locks` and `vestings`: participant's lock and vesting stake**s**

Reply:
fixed too